src/main/scripts/org/reaktivity/specification/nukleus/tcp/streams/server.sent.data/server/target.rpt

```diff
@@ -73,6 +70,3 @@ write flush
 read [0x40 0x00 0x00 0x02]
 read ${newTargetOutputEstId}
 read [0..4] # TODO: read int 13 option byteorder native
```

Comment:
Yes, I understand and agree.

There is significant flexibility in the timing and values of window updates in general, which is probably best resolved by adding a `nukleus` stream transport to k3po that handles the flow control internally for the higher level interactions and keep the low level stuff to a handful of tests that actually intend to describe how flow control should behave.